addon/components/frost-autocomplete.js

```diff
@@ -0,0 +1,125 @@
+import Ember from 'ember'
+import computed, {readOnly} from 'ember-computed-decorators'
+import {PropTypes} from 'ember-prop-types'
+import layout from '../templates/components/frost-autocomplete'
+import {keyCodes} from '../utils'
+import FrostSelect from './frost-select'
+
+const {$, get, isEmpty, run} = Ember
+const {DOWN_ARROW, UP_ARROW} = keyCodes
+
+export default FrostSelect.extend({
+  layout,
+  classNames: 'frost-select',
```

Comment:
Shouldn't this class name be `frost-autocomplete`?

Reply:
It needs this to inherit a lot of the behaviour from its parent

Reply:
I believe the question was asked because the expectation was that the `frost-select` component already has a `classNames: 'frost-select'` entry although looking at it it does not.  However, one should be added and the entry in this PR/file either removed or updated to `frost-autocomplete` if need to have specific styling not applicable to the `frost-select` component.